build-arch-gce

```diff
@@ -60,7 +60,9 @@ mount -- "$root_dev" "$mount_dir"
 
 echo '- Mounting the EFI partition.'
 mkdir -- "$mount_dir/efi"
+mkdir -- "$mount_dir/boot"
 mount -- "$efi_dev" "$mount_dir/efi"
+mount --bind -- "$mount_dir/efi" $mount_dir/boot
```

Comment:
Rather than binding the two together, I'd just mount the EFI partition on /boot.

This is the drawback of using systemd-boot though: you end up with the kernel and initramfs forced to be stored on a vfat filesystem rather than ext4.

Reply:
> Rather than binding the two together, I'd just mount the EFI partition on /boot.

I was doing that at first and then choose double mount for automatic mounting works, as mentioned here [1],

> The ESP used for the current boot is automatically mounted to /efi/ (or /boot/ as fallback), unless a different partition is mounted there (possibly via /etc/fstab, or because the Extended Boot Loader Partition — see below — exists) or the directory is non-empty on the root disk.

[1] https://systemd.io/DISCOVERABLE_PARTITIONS/